src/llvm/tools/lld/test/ELF/msp430.s

```diff
@@ -0,0 +1,13 @@
+# REQUIRES: msp430
+# RUN: llvm-mc -filetype=obj -triple=msp430-elf %s -o %t
+# RUN: llvm-mc -filetype=obj -triple=msp430-elf %S/Inputs/msp430.s -o %t2
+# RUN: ld.lld --Ttext=0x2000 %t2 %t -o %t3
+# RUN: llvm-objdump -d %t3 | FileCheck %s
+
+# R_MSP430_10_PCREL
+jmp _start
+# CHECK: jmp $-6
```

Comment:
It is not the best way to check the relocation.
`jmp $-6` by itself is not very meaningful while you do not show its address and
the address of `_start`.

We also sometimes write a comment wxplaining how the value was calculated.
So it could be something like:
```
# CHECK: Disassembly of section .text:
# CHECK-NEXT: _start:
# CHECK-NEXT:    0x2000:       {{.*}}    nop
# 0x200X + YY - 6 == 0x2000 == ADDR(.text)
# CHECK-NEXT:    0x200X:       jmp $-6  # R_MSP430_10_PCREL
```